source/mir/math/stat.d

```diff
@@ -424,6 +426,241 @@ unittest
     assert(float.max.repeat(3).hmean!(double, "fast").approxEqual(float.max));
 }
 
+/++
+Computes the median of `slice`.
+
+Can also pass a a flag `allowModify` that allows the input slice to be modified.
+By default, a copy is made. 
+
+Returns:
+    the median of the slice
+
+See_also: $(SUBREF mean)
++/
+template median(F, Flag!"allowModify" allowModify = No.allowModify)
```

Comment:
Please don't use the flags in the new code. I hope we would have named params one day.